packages/fx-core/src/plugins/resource/bot/aadRegistration.ts

```diff
@@ -72,18 +72,23 @@ export class AADRegistration {
 
   public static async registerAADAppAndGetSecretByAppStudio(
     appStudioToken: string,
-    displayName: string
+    displayName: string,
+    objectId?: string
   ): Promise<BotAuthCredential> {
     const result = new BotAuthCredential();
 
     const appConfig: IAADDefinition = {
       displayName: displayName,
     };
 
-    const app = await AppStudio.createAADAppV2(appStudioToken, appConfig);
-    result.clientId = app.appId;
-    result.objectId = app.id;
+    if (!objectId) {
+      const app = await AppStudio.createAADAppV2(appStudioToken, appConfig);
+      result.clientId = app.appId;
+      result.objectId = app.id;
+      objectId = app.id;
+    }
 
+    // if has objectId, just create password for this AAD
```

Comment:
If no appid and no objectid, create a new AAD; otherwise, the appid and objectid should both existed, and in this case, if the password is missing, create for the AAD.